stdlib/smtpd.pyi

```diff
@@ -4,6 +4,8 @@ import socket
 from collections import defaultdict
 from typing import Any
 
+__all__ = ["SMTPChannel", "SMTPServer", "DebuggingServer", "PureProxy", "MailmanProxy"]
```

Comment:
MailmanProxy is gone in 3.11 (python/cpython#26617). I'll send a separate PR for that.